### DISCUSSION TITLE
Add Counter Skin Components

### DISCUSSION THREAD

**Discussion by Rekunan** [ppy/osu#20608] [Ideas] [answered]
There'd be components in the skin layout editor for each counter in the score details: judgements (1 per), miss, sliderbreak, etc.

**Comment by mk56-spn** [comment]
I have a working prototype for this . Need to do other things so it might be a while before I finish it but it's on my list

**Reply by peppy** [reply]
if you're implementing this, make sure it's a single component with ample flexibility. See designs at https://www.figma.com/file/up6guVobzpc3DZlQxWzVmv/Gameplay-Layout?node-id=0%3A1 for reference.

**Reply by mk56-spn** [reply]
yeah i had already considered the different judgement types for mania and such

**Comment by C0D3-M4513R** [comment]
I wanted to open a new discussion, but then I found this.
In osu standard I used to use these overlays: https://github.com/l3lackShark/static

Since judgements have already have a prototype, I would like to suggest adding
some beatmap information elements, so sr, cs, ar, hp, od, difficulty name, mapper name, artist, song name and anything else that might be interesting.
Also maximum potential pp might be intresting (with a potential acc adjustment option in song-select)

My suggestions mainly come from this json api, of gosumemory: https://github.com/l3lackShark/gosumemory/wiki/JSON-values

**Reply by mk56-spn** [reply]
General beatmap stats as you are mentioning would be a separate development effort. Which is to say a different skin element . As for anyone reading this wondering why anyone would want them during gp it's pretty nifty for streamers. Might warrant it's own discussion

**Reply by C0D3-M4513R** [reply]
maybe it's time to learn c# and contribute then...
EDIT:
I wrote a skin-element for that in #21065

**Comment by mk56-spn** [comment, marked as answer]
https://github.com/ppy/osu/pull/21620 closes this.
